samples/java/hibernate/src/main/java/com/google/cloud/postgres/models/Albums.java

```diff
@@ -0,0 +1,139 @@
+package com.google.cloud.postgres.models;
+
+import com.google.cloud.postgres.CurrentLocalDateTimeGenerator;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.Lob;
+import javax.persistence.ManyToOne;
+import org.hibernate.annotations.GenerationTime;
+import org.hibernate.annotations.GeneratorType;
+import org.hibernate.annotations.Type;
+
+@Entity
+public class Albums {
+
+  @Id
+  @Column(columnDefinition = "varchar", nullable = false)
+  @GeneratedValue
+  private UUID id;
+
+  private String title;
+
+  @Column(name = "marketing_budget")
+  private BigDecimal marketingBudget;
+
+  @Column(name = "release_date", columnDefinition = "date")
+  private LocalDate releaseDate;
+
+  @Lob
+  @Type(type = "org.hibernate.type.BinaryType")
```

Comment:
Is this required? I would have expected the mapping to work by default without this